### DISCUSSION TITLE
Note cards

### DISCUSSION THREAD

**Discussion by ghost** [reuseman/flashcards-obsidian#4] [Ideas]
Thank you for a really nice and streamlined plugin for anki. I have been waiting for this!
I often create atomic notes for formulas and definitions. 
Would it be possible to create a mode, where the whole note would become a card, with the file title as the front and the entire note content as the back? Maybe by adding a tag to the front matter as a marker for the plugin to read?
Keep up the good work!

**Comment by reuseman** [comment]
@faerch90  Personally is not a thing that I use, but technically you should already be able to do it. There are two ways.

The first is by using the headings without the empty line in between. So for example
```markdown
# My huge card #card
Answer 1
## Section
Answer 2
### Another section
Another answer 
that
could
be
long
```

Or another approach could be just by creating lines to divide.
```markdown
# My huge card #card
Answer 1
---
Answer 2
---
Another answer 
```

Let me know
